HW_1/task2.py

```diff
@@ -0,0 +1,16 @@
+# Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. 
+# Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое количество журавликов, 
+# а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+# *Пример:*
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+# 60 -> 10  40  10
+
+s = int (input("Введите число "))
+rez = s % 6
+if rez == 0:
+    print ("Кол-во журавликов сделанных Петей = ", s // 6, 
+      "Кол-во журавликов сделанных Сережей =", s // 6,
+      "Кол-во журавликов сделанных Катей =" , (s // 6)*4 )
```

Comment:
А вот тут, как вы можете заметить, вы используете трижды одну и ту же операцию. Соответственно, на каждую из этих операций выделяются ресурсы вашей вычислительной машины. Чтобы этого не происходило, принято использовать переменные.
```suggestion
    var = s // 6
    print ("Кол-во журавликов сделанных Петей = ", var, 
      "Кол-во журавликов сделанных Сережей =", var,
      "Кол-во журавликов сделанных Катей =" , var * 4 )
```